patches/metro-react-native-babel-preset+0.66.2.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/node_modules/metro-react-native-babel-preset/src/configs/main.js b/node_modules/metro-react-native-babel-preset/src/configs/main.js
```

Comment:
for some reason excluding the plugin in the babel config for the preset does not work, there are some open issues so for now I adding a patch to remove it